README.md

```diff
@@ -4,6 +4,8 @@ This README is an index of [design docs](designdocs.md) for important components
 ### Design Docs
 _Newest to oldest_
 
+* [Model for change: Filesystem repo migrations](https://docs.google
+.com/document/d/1THzh1mrNCKYbdk1zP72xV8pfr1yQBe2n3ptrSAYyVI8/edit?usp=sharing) (In review) - April 2019
```

Comment:
This link currently only has view access turned on. Could you enable public comments?(instructions here: https://github.com/filecoin-project/designdocs/blob/master/designdocs.md#design-doc-process)

Reply:
Weird, I thought I'd enabled it since it said "anyone with the link can comment"; let me try again.